README.md

```diff
@@ -64,25 +53,11 @@ cmake --build . --target install
 ```
 
 #### Windows
-
 ``` sh
-# Create a workspace directory to hold all the SDK files
```

Comment:
we probably don't need to go so far as instructing someone to create a workspace directory though. I think the use of the word "workspace" could cause confusion between this directory and the "absolute path to sdk-workspace" since I don't believe they're both pointing to the same directory.